docs/explanation/core/configuration.md

```diff
@@ -632,18 +631,21 @@ What happens in reality is, however, a little bit different:
 #1 block - Bob
 #2 block - Bob
 #3 block - Bob
-#4 block - Alice
+#4 block - Bob
 ```
 
-That's because Bob is too fast at proposing blocks. This leaves Alice very
-little chances to propose a block and not always be catching up. Note that every
-block Bob creates needs a vote from Alice to constitute 2/3+.
+That's because Bob doesn't wait for a proposal from Alice (prevotes `nil`) and
+always moves onto the next round (NOT height). This leaves Alice no chances to
```

Comment:
I would remove the second part of this sentence: "and always moves onto the next round (NOT height)".

Both move to the next round when prevotes are received but do not form a quorum (as Bob prevoted nil), so that both precommit nil, thus eventually moving to the next round. This is a slow path because, in addition to receiving all the votes, they have to wait for two timeouts (prevote and precommit).